perfdash/config.go

```diff
@@ -666,15 +666,33 @@ var (
 		},
 	}
 
+	benchmarkListDescription = TestDescriptions{
+		"E2E": {
+			"Resources": []TestDescription{{
+				Name:             "List benchmark",
```

Comment:
All the places seems to use name with different-no-space-naming-schema. You are including spaces etc. It's not clear to me if this will not cause any problems.